### DISCUSSION TITLE
error 400: redirect_uri_mismatch

### DISCUSSION THREAD

**Discussion by letran3691** [outline/outline#2731] [Self Hosting]
I try deploy self-hosted and using reverse proxy nginx. When I login with google I seem "error 400: redirect_uri_mismatch". 
![image](https://user-images.githubusercontent.com/19284401/140254350-19b4cbcc-bb37-4d00-a676-30f22b18b212.png)

file env

![image](https://user-images.githubusercontent.com/19284401/140254425-f0e70fcb-0498-49b6-b899-b2985d33ae6a.png)

I don't know reason
someone help me. thank.

**Comment by letran3691** [comment]
[Update]
I resolved that issue. But I've other issue
![image](https://user-images.githubusercontent.com/19284401/140268646-c9d8c979-4cdf-4b98-a2a0-a2c0aa46d8fa.png)

**Reply by tommoor** [reply]
It means you have set the `ALLOWED_DOMAINS` environment variable but it doesn't match the account you're attempting to connect

**Reply by letran3691** [reply]
I've issue with S3 
![image](https://user-images.githubusercontent.com/19284401/140457170-28fe3c71-f66f-49d9-82a2-0ea435500b2f.png)

ACCESS KEY and SECRET  KEY is group admin and owner bucket, but access denied 
 ![image](https://user-images.githubusercontent.com/19284401/140457295-dd8941d7-05fa-4eef-a53f-12c48a431d59.png)

**Comment by tommoor** [comment]
Please run through the [S3 setup guide](https://wiki.generaloutline.com/share/125de1cc-9ff6-424b-8415-0d58c809a40f).

**Reply by letran3691** [reply]
today I'm tries config login via google but dashboard can't display option login with google 
![image](https://user-images.githubusercontent.com/19284401/140475318-5607316e-876d-424d-8530-4ce7555d89de.png)
![image](https://user-images.githubusercontent.com/19284401/140475574-18d2cb0c-3dcf-4e01-84a4-af528bc33717.png)
![image](https://user-images.githubusercontent.com/19284401/140475369-f50b5383-8630-43f6-8bfb-8699cbc2fdd3.png)
